src/backend/src/controllers/work-packages.controllers.ts

```diff
@@ -103,6 +103,13 @@ export const createWorkPackage = async (req: Request, res: Response) => {
 
   if (workPackageNumber !== 0) throw new TypeError('Given WBS Number is not for a project.');
 
+  if(dependencies.find((dep: { carNumber: any; projectNumber: any; workPackageNumber: any; }) =>
+    dep.carNumber === carNumber
```

Comment:
I'm pretty sure I wrote a utility WBS Number equals function already

Reply:
Changed it to use existing.